### ISSUE TITLE
TV Buy advice VS current price

### ISSUE THREAD

**Issue by BramFr** [chrisleekr/binance-trading-bot#385]
I`m still investigation about TV Buy/Sell advice

But if ill understand correctly based on the picture below. Its interesting to buy this coin at the max price 0.014994 and sell when the price around 0.015904.

The current price 0.015144 is between those advises but still higher then the tv buy advice price. Because the TV has a buy advice the bot will place a buy order when possible.

im more satisfied to place a buy order when the current price is below 0.014994 ore is there something i missed ?
![image](https://user-images.githubusercontent.com/16082279/139679874-23d56f15-c9d0-4067-8909-e06a8e04bc9a.png)

**Comment by chrisleekr** [comment]
@BramFr You seems not using the latest TradingView that getting all TradingView indicators per tick. Which version are you using?

Try to use the latest version (v0.0.85), which refactored TradingView to get all symbols Technical Analysis at once.

**Comment by BramFr** [comment]
> @BramFr You seems not using the latest TradingView that getting all TradingView indicators per tick. Which version are you using?
> 
> Try to use the latest version (v0.0.85), which refactored TradingView to get all symbols Technical Analysis at once.

Thanks updated to 85. 
But this version does have the same question :)
Another example:

This one hase advice Strong Buy.. 
But the current price is higher then the "Moving Average" advice.
At my opinion the current price must below the (Simple Moving Average 10) price.

Correct me if im wrong but the number 10/20/30/40/ ... are the percent difference between the current price. When you have a sell triger of 1.18 (18%). The safest way is to look at the Tv Advice (Simple Moving Average **20**).
With this exmaple its safe to buy when the price drops to 185.23
![image](https://user-images.githubusercontent.com/16082279/139833583-7988737f-beca-43a6-9100-1cc55976e391.png)

**Comment by Rayn0r** [comment]
As far as I understood, the MA(moving average) is the mean value of the closing price over certain period of time.
So to calculate SMA(10), you sum up the coin value at the end of each day for the past 10 days and divide it by 10...

**Comment by chrisleekr** [comment]
@BramFr Yeah, @Rayn0r is correct. You are even looking at "Exponential Moving Average". 
Since you are using 1d interval, the EMA is for the last 10 days.

**Comment by chrisleekr** [comment]
Closing this as a refactored v1 has been released. It's no longer relevant to the latest code.
